Formula/dgraph.rb

```diff
@@ -17,7 +17,7 @@ class Dgraph < Formula
     sha256 cellar: :any_skip_relocation, mojave:         "57fa2f974e4e0313fb58f7e35e2d9547a4d43319af7e84a4fc6619238e862885"
   end
 
-  deprecate! date: "2021-06-10", because: :unsupported
+  disable! date: "2022-06-17", because: :unsupported
```

Comment:
I wonder if we should make try to make this Linux-only & update to latest given this was deprecated due to upstream dropping macOS support.

Though, there may be an issue with behavior of macOS users who installed existing bottle. I think we saw a similar issue when `at-spi2-core` was resurrected as Linux-only.